tests/ConceptTest.php

```diff
@@ -459,6 +452,7 @@ public function testGetGroupPropertiesWithHierarchy()
      */
     public function testGetPropertiesWithNarrowersPartOfACollection()
     {
+        $this->markTestSkipped('disabled since the functionality needs to be reimplemented after the new translation component is in use');
```

Comment:
Same as above. Don't mark the test as skipped anymore.

Reply:
Removed in new commit: https://github.com/NatLibFi/Skosmos/commit/48b31a51acb2fd8be2452c033e0cecd05d86c764